docker/docker-compose.1604.51.yaml

```diff
@@ -0,0 +1,42 @@
+version: "3"
+
+services:
+
+  runtime-setup:
+    image: swift-nio:16.04-5.1
+    build:
+      args:
+        ubuntu_version: "16.04"
```

Comment:
why not 18.04?

Reply:
Heh, [great minds](https://github.com/apple/swift-nio/pull/991#issuecomment-488660476)!